### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by KaratekHD** [KaratekHD/defrag#5]

**Comment by why-not-try-calmer** [comment]
Yes we don't have 100% test coverage yet. I'll make some more rounds to little by little get us there.

**Comment by why-not-try-calmer** [comment]
I think we have enough tests for now to make the testing workflow a non-issue.
